.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,14 +1,14 @@
 name: Feature request
-description: Feature request template.
-title: '[Feature]: '
-labels: ['feature']
+description:  "Suggest features, propose improvements, discuss new ideas"
+title: '[FEATURE]: '
+labels: ['enhancement']
 body:
 - type: markdown
     attributes:
       value: '## This issue template is not for requesting new algorithms. For new algorithms, PRs should be opened directly.'
```

Comment:
This is dirty.

Reply:
@appgurueu What do you mean by "dirty"?

Reply:
You're using a Markdown field - which will often end up being submitted as part of the feature request - just to inform the user of something.

Reply:
I am not sure I see the problem. According to [the documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown):

> You can use a markdown element to display Markdown in your form that provides extra context to the user, but is not submitted.

So it seems like a perfect match for this use case. @appgurueu

Reply:
> You're using a Markdown field - which will often end up being submitted as part of the feature request - just to inform the user of something.

Nope. That is removed once the issue is submitted and is only available to the user _before_ submitting.